llvm/lib/Transforms/PeepholeOptimizationCourse/ObfuscateI8AddFunctions/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_llvm_component_library(
+    LLVMObfuscateI8AddFunctions
```

Comment:
I don't see where you link against that library. How did linker find the implementation?